Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8192,3 +8192,17 @@ def get_tenant_account_name():
         account_name = "acc_{}".format(tenant_name) if tenant_name != "" else ""
 
     return account_name
+
+def indicators_value_to_clickable(indicators):
+    if not isinstance(indicators, list):
+        indicators = [indicators]
```

Comment:
add a condition for None.
Please also add docstring.

Reply:
return an empty dict in case of None?